packages/steps/src/BuildStep.ts

```diff
@@ -217,7 +217,9 @@ export class BuildStep {
     const newPath = currentPath ? `${BIN_PATH}:${currentPath}` : BIN_PATH;
     return {
       ...env,
+      __EXPO_STEPS_BUILD_ID: this.ctx.buildId,
```

Comment:
Not a part of this PR, but if we are passing more and more values, maybe it would make sense to use sth different than envs, so it's not visible to users?

Reply:
Any suggestions?

Reply:
-generate scripts with embeded values
-settings those values in file in some known location

Reply:
>Not a part of this PR, but if we are passing more and more values, maybe it would make sense to use sth different than envs, so it's not visible to users?

I agree that this would be good, although I don't see a better option at the moment. I'm also curious what's the @wkozyra95 suggestion.